pkg/util/wait/wait_test.go

```diff
@@ -173,8 +173,25 @@ func TestJitterUntilNegativeFactor(t *testing.T) {
 }
 
 func TestExponentialBackoff(t *testing.T) {
+	sleeps := []time.Duration{}
+	sleepFn = func(d time.Duration) { sleeps = append(sleeps, d) }
+	defer func() { sleepFn = time.Sleep }()
+
 	opts := Backoff{Factor: 1.0, Steps: 3}
 
+	expectSleeps := func(expected ...time.Duration) error {
+		defer func() { sleeps = nil }()
```

Comment:
I think personally I'd probably make a test fixture and separate tests rather than clear this every time, but I don't care that much.